h2o-py/h2o/__init__.py

```diff
@@ -26,19 +26,30 @@
 from h2o.frame import H2OFrame  # NOQA
 from h2o.utils.shared_utils import mojo_predict_csv, mojo_predict_pandas
 
+import zipfile
 import os
 from codecs import open
 here = os.path.abspath(os.path.dirname(__file__))
 
 try:
-    with open(os.path.join(here, 'buildinfo.txt'), encoding='utf-8') as f:
-        __buildinfo__ = f.read()
+    if here.endswith('.whl/h2o'):
```

Comment:
there is no guaratee that here.endswith('.whl/h2o') means .whl is a zipfile, I see two options
1) make this a fallback when the original open() fails 
2) add check that the parent is whl and it is in fact a regular file not a dir
i think 1) is better